src/services/directory.ts

```diff
@@ -6,10 +6,11 @@
  */
 
 import { backendFetchJson, getRequestParamFromList } from './utils';
+import { UnknownArray } from 'type-fest';
 
 const PREFIX_DIRECTORY_SERVER_QUERIES = import.meta.env.VITE_API_GATEWAY + '/directory';
 
-export function fetchContingencyAndFiltersLists(listIds) {
+export function fetchContingencyAndFiltersLists(listIds: UnknownArray) {
```

Comment:
UUID [] ? or at least a string[]

Reply:
That's what it's used in the reducer..  [PARAM_FAVORITE_CONTINGENCY_LISTS]: UnknownArray;
Check the use of fetchContingencyAndFiltersLists.